docker/docker-compose-base.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, EMC, Inc.
+
+version: '2'
+
+volumes:
+  dhcp-leases:
+    external: false
+
+services:
```

Comment:
I would guess the requirement only expect RackHD-only services, but that does not seem explain why mongo service is presented here.